docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |edit the detail of a person
+ |easily make change to their detail when they updated their contact
+
+|`* * *` |user |delete a person |remove entries that I no longer need
+
+|`* * *` |user |create a class |group and manage students who are taking the same class
+
+|`* * *` |user |add persons to a class |
+
+|`* * *` |user |see all persons in a class |see the list of all students taking a class
```

Comment:
teacher | list all the students in a particular class | know all the students taking that class